src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const trs = document.querySelectorAll('tr');
+
+trs.forEach((el) => {
+  const parentElement = el;
+
+  if (parentElement.children.length > 1) {
+    const firstChild = parentElement.children[1];
+    const clonedChild = firstChild.cloneNode(true);
+
+    parentElement.insertBefore(
+      clonedChild,
+      parentElement.children[parentElement.children.length - 1],
```

Comment:
The task specifies appending the cloned column in front of the last column, but the code currently inserts it before the last child of each row. Make sure to adjust the logic to append the cloned column correctly in front of the last column.